randovania/games/prime1/exporter/scan_tips.py

```diff
@@ -0,0 +1,696 @@
+            "By using another double bomb jump on top of it, "
+            "Great Tree Chamber is reachable from Great Tree Hall without Space Jump Boots."
+        ),
+    ],
+    "2541494929": [
+        # Vault Jump Tip
+        # Replaces Main Plaza fallen stones scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;One can reach the ledge leading to Vault with "
+            "&main-color=#FF3333;Space Jump Boots&main-color=#89D6FF;.\n\nIt requires holding "
+            "&image=SA,3.0,1.0,1.0,46434ED3,34E79314; while jumping from a brick platform under a twisting tree."
+        ),
+    ],
+    "2903166635": [
+        # Metroid Prime Flamethrower Exo Tip
+        # Replaced Metroid Prime Exo scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "Metroid Prime",
+        (
+            "&main-color=#89D6FF;When Metroid Prime's exoskeleton is weak to the Power Beam, "
+            "it can also be damaged by the &main-color=#FF3333;Flamethrower&main-color=#89D6FF;."
+        ),
+    ],
+    "2271263048": [
+        # Hive Mecha Bridge Tip
+        # Replaces Hive Mecha bridge scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        "&main-color=#89D6FF;To cross the bridge without fighting Hive Mecha, carefully walk along its right edge.",
+    ],
+    "2053272936": [
+        # Boost Pipe Speed Tip
+        # Replaces Tallon Canyon halfpipe scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;When using the &main-color=#FF3333;Boost Ball&main-color=#89D6FF; in a half pipe, "
+            "hold only &image=8B0C22A7; or &image=778CCD6A; and boost at the lowest point to gain the most height."
+        ),
+    ],
+    "399696251": [
+        # Boost Speed Limit Tip
+        # Replaces Ruined Gallery morph tunnel scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;When using the &main-color=#FF3333;Boost Ball&main-color=#89D6FF;, "
+            "if the control stick is kept in a neutral position, the Boost Ball has no speed limit."
+        ),
+    ],
+    "4175379874": [
+        # Flaahgra Wavebuster Tip
+        # Replaced Flaahgra scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "Flaahgra",
+        (
+            "&main-color=#89D6FF;Flaahgra can be defeated with the &main-color=#FF3333;Wavebuster&main-color=#89D6FF;. "
+            "On Normal difficulty, doing so without any other sources of damage requires 230 missiles."
+        ),
+    ],
+    "889553182": [
+        # Frozen Metroid Hop Tip
+        # Replaces Metroid scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "Metroid",
+        (
+            "&main-color=#89D6FF;Jumping on the center of a frozen Metroid causes one to bounce on top of it.\n\n"
```

Comment:
This seems unnecessary. This information isn't useful for players who don't have this trick in logic; those who do have it in logic very likely already know how it works. Most players will also be focused on killing the metroids instead of scanning them, so the likelihood they would see this scan and find it useful is unlikely.

Reply:
This falls within my third goal of explaining unintuitive mechanics, and I also just disagree that players who set high difficulties will know about all the tricks that fall within them. There's not much reason to assume there's something special about standing on frozen Metroids as a trick - I would have just assumed the trick difficulty was so high because manipulating them into a good position is hard. Plus, a player might read this, find the idea interesting, and then want to try it. I don't think every tip needs to be for the purpose of helping a player follow the logic of their seed - educating players on niche mechanics and tacitly inviting them to try weird stuff like the Metroid hop out is enough.